MAUI/Autocomplete/Selection.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: Selection in .NET MAUI Autocomplete control | Syncfusion
+description: Learn here all about Selection support in Syncfusion .NET MAUI Autocomplete (SfAutocomplete) control and more.
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---
+
+# Selection in .NET MAUI Autocomplete (SfAutocomplete)
+
+The `Autocomplete` allows user to select a single item from dropdown list.
+
+## Single selection
+
+The `Autocomplete` allows user to select an item by entering the value using keyboard, then selecting from the drop-down list by clicking the item. The selected item can be retrieved from the `SelectedItem` property.
```

Comment:
The `Autocomplete` allows the user to select an item by entering the value using the keyboard, then selecting from the drop-down list by clicking the item. The selected item can be retrieved from the `SelectedItem` property.